### PR TITLE
Use CircleCI rather than Travis

### PR DESCRIPTION
**Pull request by symroe**
It's generally nicer, but also the build is now <2 minutes vs who knows how long on Travis.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v2-dependencies-{{ checksum "requirements/base.txt" }}-{{ checksum "requirements/testing.txt" }}
```
Comment (chris48s) [inline .circleci/config.yml]: :tada:

**Comment by symroe** [comment]
Nice one, I forgot we were still on 9.5. Downgraded the docker image.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
+      - image: circleci/python:3.6.1
```
Comment (chris48s) [inline .circleci/config.yml]: also the python version on prod will be 3.5

File: tox.ini [hunk 3]
```diff
@@ -1,12 +1,9 @@
 [tox]
 # Ensure you add to .travis.yml if you add here
 # envlist = {py27,py34}-django{18,19,110}
-envlist = py{35}-django111
+envlist = py{36}-django111
```
Comment (chris48s) [inline tox.ini]: ..and here

**Pull request merged**